lib/util/sdp/simulcast.js

```diff
@@ -109,10 +110,15 @@ class TrackAttributes {
     const simSSRCs = Array.from(this.primarySSRCs.values());
     const ssrcs = rtxPairs.length ? flatMap(rtxPairs) : simSSRCs;
 
+    const msidAttrPrefix = this.streamId
+      ? `${this.streamId} `
+      : '';
+
     const attrLines = flatMap(ssrcs, ssrc => [
       `a=ssrc:${ssrc} cname:${this.cName}`,
-      `a=ssrc:${ssrc} msid:${this.streamId} ${this.trackId}`
+      `a=ssrc:${ssrc} msid:${msidAttrPrefix}${this.trackId}`
```

Comment:
I do not think this is valid. If there is no MediaStream associated with the MediaStreamTrack, then we should stamp "-". Per [JSEP](https://tools.ietf.org/html/draft-ietf-rtcweb-jsep-24),

```
      *  If no MediaStream is associated with the transceiver, a single
         "a=msid" line with the special value "-" in place of the
         MediaStream ID, as specified in [I-D.ietf-mmusic-msid],
         Section 3.  The track ID MUST be selected as described above.
```

Also, rather than `msidAttrPrefix`, we could call this something like `msidId` to better match the definition in [draft-ietf-mmusic-msid-16](https://tools.ietf.org/html/draft-ietf-mmusic-msid-16#section-2).